contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -345,20 +351,24 @@ function Package-OpenSSH
             New-Item -ItemType Directory $DestinationPath -Force | Out-Null
         }
         Copy-Item -Path $packageDir\* -Destination $DestinationPath -Force -Recurse
+        Write-Host 'Copied payload to' $DestinationPath
     }
     else {
         Remove-Item ($packageDir + '.zip') -Force -ErrorAction SilentlyContinue
         Compress-Archive -Path $packageDir -DestinationPath ($packageDir + '.zip')
+        Write-Host 'Packaged Payload - '$packageDir'.zip'
```

Comment:
Write-BuildMsg -AsInfo -Message "Packaged Payload - '$packageDir'.zip". The message will write to host and log. The apply to below multiple places